typespec-tests/src/test/java/com/type/property/additionalproperties/ExtendsTests.java

```diff
@@ -65,13 +65,16 @@ public void testExtendsModelArrayClient() {
     public void testExtendsModelClient() {
         ExtendsModelAdditionalProperties body = new ExtendsModelAdditionalProperties();
         Map<String, ModelForRecord> propertyMap = new LinkedHashMap<>();
+        propertyMap.put("knownProp", new ModelForRecord("ok"));
```

Comment:
"knownProp" should not be set via additionalPorperties, and should not be retrieved from additionalPorperties

If it cannot be set/get via model, disable the test, mark it as a bug.

Reply:
Disabled this test case, and `IsModelArrayClientTest#testPullAndGet` and `IsModelClientTest#testPullAndGet` same as this.